_posts/2020-07-06-release-1.11.0.md

```diff
@@ -0,0 +1,307 @@
+
+**Docker Image Unification ([FLIP-111](https://cwiki.apache.org/confluence/display/FLINK/FLIP-111%3A+Docker+image+unification))**
+
+With this release, all Docker-related resources have been consolidated into [apache/flink-docker](https://github.com/apache/flink-docker) and the entry point script has been extended to allow users to run the default Docker image in different modes without the need to create a custom image. The [updated documentation](https://ci.apache.org/projects/flink/flink-docs-master/ops/deployment/docker.html#customize-flink-image) describes in detail how to use and customize the official Flink Docker image for different environments and deployment modes.
+
+<hr>
+
+### Table API/SQL: Support for Change Data Capture (CDC)
+
+Change Data Capture (CDC) has become a popular pattern to capture committed changes from a database and propagate those changes to downstream consumers, for example to keep multiple datastores in sync and avoid common pitfalls such as [dual writes](https://thorben-janssen.com/dual-writes/). Being able to easily ingest and interpret these changelogs into the Table API/SQL has been a highly demanded feature in the Flink community — and it’s now possible with Flink 1.11.
+
+To extend the scope of the Table API/SQL to use cases like CDC, Flink 1.11 introduces new table source and sink interfaces with **changelog mode** (see _[New TableSource and TableSink Interfaces](#other-improvements-to-the-table-apisql)_) and support for the [Debezium](https://debezium.io/) and [Canal](https://github.com/alibaba/canal) formats ([FLIP-105](https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=147427289)). This means that dynamic tables sources are no longer limited to append-only operations and can ingest these external changelogs (``INSERT`` events), interpret them into change operations (``INSERT``, ``UPDATE``, ``DELETE`` events) and emit them downstream with the change type.
+
+<center>
+	<figure>
+	<img src="{{ site.baseurl }}/img/blog/2020-07-06-release-1.11.0/image4.png" width="500px" alt="CDC"/>
+	</figure>
+</center>
+
+<div style="line-height:150%;">
+    <br>
+</div>
+
+Users have to specify either ``“format=debezium-json”`` or ``“format=canal-json”`` in their ``CREATE TABLE`` statement to consume changelogs using SQL DDL. 
+
+```sql
+CREATE TABLE my_table (
+  ...
+) WITH (
+  'connector'='...', -- e.g. 'kafka'
+  'format'='debezium-json',
+  'debezium-json.schema-include'='true' -- default: false (Debezium can be configured to include or exclude the message schema)
+  'debezium-json.ignore-parse-errors'='true' -- default: false
+);
+```
+
+Flink 1.11 only supports Kafka as a changelog source out-of-the-box and JSON-encoded changelogs, with Avro (Debezium) and Protobuf (Canal) planned for future releases. There are also plans to support MySQL binlogs and Kafka compacted topics as sources, as well as to extend changelog support to batch execution.
+
```

Comment:
As discussed in the RC4 voting thread, the the critical issue FLINK-18461 of CDC feature will not block the release. We may need to add a note in the release announcement for this. We can remove the note in the next 1.11.1 version, or in the next RC if we have. 

What do you think about this passage? 

```
<span class="label label-danger">Attention</span> There is a known issue ([FLINK-18461](https://issues.apache.org/jira/browse/FLINK-18461)) that changelog source can't be INSERT INTO upsert sinks (e.g. MySQL, HBase, Elasticsearch). This will be fixed in the next 1.11.1 version.
```

![image](https://user-images.githubusercontent.com/5378924/86525077-bf3c1d00-beb4-11ea-9958-9689b5187a60.png)

Reply:
Thanks for pointing it out, Jark! I’ll add the note.

Reply:
Thanks @morsapaes !